### PR TITLE
add a section for openstack

### PR DESCRIPTION
**Pull request by weshayutin**
## Why the changes were made

Because I promised to update the docs for openstack e2e tests 

## How to test the changes made

you know the drill, read the docs

### CODE REVIEW

File: docs/developer/testing/TESTING.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ OPENSHIFT_CI=false \
 make test-e2e
 ```
 
+### OpenStack setup
+
+* no changes needed, just setup s3 storage in aws.
+
```
Comment (mateusoliveira43) [inline docs/developer/testing/TESTING.md:62]: Add info here
https://github.com/openshift/oadp-operator/blob/510c2033cbbae5cbd207da04d99490bd42c41466/docs/developer/testing/TESTING.md?plain=1#L11

not in a new section

Comment (weshayutin) [reply docs/developer/testing/TESTING.md:62]: ok.. added OpenStack in the section you mentioned as well but also kept the openshtack section to make it more obvious.

File: docs/developer/testing/TESTING.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ You need to have OADP operator already installed in your cluster to run E2E test
 
 ### AWS setup
 
-> **Note:** If you are using IBM Cloud, follow this section.
+> **Note:** If you are using IBM Cloud, or OpenStackfollow this section.
```
Comment (kaovilai) [inline docs/developer/testing/TESTING.md:11]:
```suggestion
> **Note:** If you are using IBM Cloud, or OpenStack follow this section.
```

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by kaovilai — APPROVED** [review]
/lgtm

**Pull request merged**